storage/storage_base.hpp

```diff
@@ -49,7 +51,9 @@ namespace bzn
         {storage_result::db_not_found,    "DATABASE_NOT_FOUND"},
         {storage_result::db_exists,       "DATABASE_EXISTS"},
         {storage_result::access_denied,   "ACCESS_DENIED"},
-        {storage_result::delete_pending,  "DELETE_PENDING"}};
+        {storage_result::delete_pending,  "DELETE_PENDING"},
+        {storage_result ::db_create_denied, "DB_CREATE_DENIED"},
```

Comment:
We don't need more types. ACCESS_DENIED is good enough.

Reply:
OK, removing DB_CREATE_DENIED and DB_DELETE_DENIED, using ACCESS_DENIED instead.